.goreleaser.docker.yaml

```diff
@@ -280,6 +303,29 @@ dockers:
       - '--label=org.opencontainers.image.title=jvs-ui'
       - '--label=org.opencontainers.image.version={{ .Version }}'
 
+  -
+    ids:
```

Comment:
Not that familiar with goreleaser. But wondering should we add this into the original jvs go releaser yaml?